[2022-12-11 18:02:18,955] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task scheduled__2022-12-11T17:59:00+00:00 [queued]>
[2022-12-11 18:02:18,995] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task scheduled__2022-12-11T17:59:00+00:00 [queued]>
[2022-12-11 18:02:18,999] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 18:02:19,002] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 18:02:19,004] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 18:02:19,045] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): write_diff_task> on 2022-12-11 17:59:00+00:00
[2022-12-11 18:02:19,059] {standard_task_runner.py:52} INFO - Started process 4059 to run task
[2022-12-11 18:02:19,078] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'write_diff_task', 'scheduled__2022-12-11T17:59:00+00:00', '--job-id', '1150', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpr_31trf_', '--error-file', '/tmp/tmp3hpywwep']
[2022-12-11 18:02:19,081] {standard_task_runner.py:80} INFO - Job 1150: Subtask write_diff_task
[2022-12-11 18:02:19,222] {task_command.py:369} INFO - Running <TaskInstance: first_dag.write_diff_task scheduled__2022-12-11T17:59:00+00:00 [running]> on host 7e39590a7807
[2022-12-11 18:02:19,443] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=write_diff_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T17:59:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-11T17:59:00+00:00
[2022-12-11 18:02:19,515] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 18:02:19,555] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=write_diff_task, execution_date=20221211T175900, start_date=20221211T180218, end_date=20221211T180219
[2022-12-11 18:02:19,614] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 18:02:19,719] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
